main/src/main/java/org/mobilitydata/gtfsvalidator/notice/StartAndEndDateOutOfOrderNotice.java

```diff
@@ -19,6 +19,11 @@
 import com.google.common.collect.ImmutableMap;
 import org.mobilitydata.gtfsvalidator.type.GtfsDate;
 
+/**
+ * Two {@code GtfsDate} are out of order
+ *
+ * <p>Severity: {@code SeverityLevel.WARNING}
```

Comment:
I think we can safely call this an ERROR. feed_info.txt says:

> The feed_end_date date must not precede the feed_start_date date if both are given. 

It's not explicitly listed for calendar.txt, but it doesn't make sense to be to split off separate WARNINGs for that file.

@MobilityData/transit-specs?

Reply:
I agree, it doesn't logically make sense. This can be treated as an ERROR.
Registered in Discrepancies Specification vs Validator https://github.com/MobilityData/gtfs-tasks/issues/87